reconcile/utils/ocm.py

```diff
@@ -602,6 +602,8 @@ def __init__(
         blocked_versions=None,
         ocm_client: Optional[OCMBaseClient] = None,
         sectors: Optional[list[dict[str, Any]]] = None,
+        inheritVersionData: Optional[list[dict[str, Any]]] = None,
+        publishVersionData: Optional[list[dict[str, Any]]] = None,
```

Comment:
i'm not a huge fan of unstructured list-dicts but i get that it would be cumbersome to come up with a dataclass just to represent this and using `qenerate` would be a bigger refactor.

idea: simplify to `list[str]` that holds the names of the OCM orgs to inherit from and publish to and verify at the place where the `OCM` object is initialized, that inherit and publish are consistent.

feel free to ignore.

Reply:
I wanted to keep the whole OCM info in order to have access to other fields if needed. Though it is not needed today. Will think through this.

Reply:
ah understood. good point.